.github/workflows/build.yaml

```
@@ -0,0 +1,43 @@
+name: Build & Test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  doit:
+    name: 'Build & Test: ${{matrix.cfg.name}}'
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - cfg: {name: 'macos',             runson: 'macos-latest'}
+          - cfg: {name: 'gcc10',             runson: 'ubuntu-latest', container: 'debian:bullseye'}
+          - cfg: {name: 'gcc11',             runson: 'ubuntu-latest', container: 'ubuntu:jammy'}
+          - cfg: {name: 'gcc12',             runson: 'ubuntu-latest', container: 'debian:bookworm'}
+          - cfg: {name: 'gcc-arch',          runson: 'ubuntu-latest', container: 'archlinux'}
+          - cfg: {name: 'clang-arch',        runson: 'ubuntu-latest', container: 'archlinux',
+                  cmake: '-DCMAKE_CXX_COMPILER=clang++'}
+          - cfg: {name: 'clang-libc++-arch', runson: 'ubuntu-latest', container: 'archlinux',
+                  cmake: '-DCMAKE_CXX_COMPILER=clang++ -DCMAKE_CXX_FLAGS=-stdlib=libc++'}
```

do you need quotes here, as in `
```suggestion
                  cmake: '-DCMAKE_CXX_COMPILER=clang++ -DCMAKE_CXX_FLAGS="-stdlib=libc++"'}
```

---

I don't believe so: there isn't anything there that terminates the input (like a space would). It's easy enough to empirically test,
```
$ cmake -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_CXX_FLAGS=-stdlib=libc++ ..
...
$ make VERBOSE=1
...
[  7%] Building CXX object CMakeFiles/bls12-381.dir/src/arithmetic.cpp.o
/usr/bin/clang++  -I/home/spoon/f/leap.play/libraries/libfc/libraries/bls12-381/include -stdlib=libc++ -O3 -DNDEBUG -std=gnu++20 -MD -MT CMakeFiles/bls12-381.dir/src/arithmetic.cpp.o -MF CMakeFiles/bls12-381.dir/src/arithmetic.cpp.o.d -o CMakeFiles/bls12-381.dir/src/arithmetic.cpp.o -c /home/spoon/f/leap.play/libraries/libfc/libraries/bls12-381/src/arithmetic.cpp
...
```